Simonyi/Non-Standard-Operations/ComCam/comcam-cooldown.rst

```diff
@@ -34,14 +34,14 @@ Overview
 
 This procedure describes the starting up and cooling down of the ComCam cryostat.
 
-.. _ComCam-Cooldown-Prerequisites:
+.. _ComCam-Cooldown-Precondition:
 
-Prerequisites
+Precondition
 ^^^^^^^^^^^^^
```

Comment:
^^^^ --> ====?

Reply:
Several more of this outdated second-level headings have been modified.